gnocchi/tests/functional/gabbits/aggregates-with-metric-ids.yaml

```diff
@@ -774,8 +780,11 @@ tests:
       data:
         operations: "(rolling blah 2 (metric $HISTORY['create metric1'].$RESPONSE['$.id'] mean))"
       status: 400
-      response_strings:
-        - "Invalid input: 'rolling' operation invalid for dictionary value"
+      response_json_paths:
+        $.description.cause: "Invalid input"
+        # Use regex to avoid the 'u' populated by voluptuous
```

Comment:
not entire sure what this comment is about? something about unicode? i would remove the copy/paste but not a big deal.

Reply:
You are right, it's about the unicode: the sentences contain 'u' in python2. :roll_eyes: 

I removed the comments and mention this in a commit message. ^^